schema_based_on_diagram.sql

```diff
@@ -75,8 +75,15 @@ CREATE INDEX ON medical_histories (treatment_id);
 ALTER TABLE medical_histories DROP CONSTRAINT fk_patients;
 ALTER TABLE medical_histories DROP CONSTRAINT fk_treatments;
 
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229874645-ff60c1c8-4406-405f-bb16-8835cb127a5e.png)

**[OPTIONAL]**

It would be very great if you make all your commit messages more meaningful and capitalize them because I saw your commit is not capitalized and meaningful enough 😨 Kindly consider them making capitalized and meaningful to make your pull request professional 👍

Reply:
Sorry for that @HAMMAS-SALEEM I will make better commit messages next time.